README.md

```diff
@@ -44,49 +44,48 @@ Your code interview will be evaluated based on your repo, so make sure all files
 
 We love the NYC 311 service and the open data products that come with it. In this challenge, you will use **[311 Service Requests from 2010 to Present](https://data.cityofnewyork.us/Social-Services/311-Service-Requests-from-2010-to-Present/erm2-nwe9)** on NYC Open Data, write an ETL pipeline, and produce some data insight.
 
-### Task 1: Data Download
+This challenge has two halves, a python/bash half with 4 steps and a docker/SQL part with 2 steps. The second half will use a file generated by the first python step. It's our expectation that the logic to perform each step in the first half of this challenge will be written in python, and be callable by an executable script that takes specified arguments as inputs.
 
-Write a python script/notebook to download all service request records created in the **last week** (7 days) and has **HPD** as the responding agency, and store the data in a csv named `raw.csv` in a folder called `data`.
+### Python Task 1: Data Download
 
-### Task 2: Data Aggregation
+To download 311 service request records write a script that takes 2 parameters passed from the command line: number of days and responding agency acronym. For example, if a user wanted to get all service request records created in the last week where DSNY is the responding agency, they would pass `7` and `DSNY` as the parameters. For this exercise, we ask that you download all 311 service requests filed the **last seven days** where **HPD** is the responding agency.  Save the data as a csv named `raw.csv` in a folder called `data`. 
 
-Create a time series table based on the `data/raw.csv` file we created from **Task 1** that has the following fields
+### Python Task 2: Data Aggregation
 
-- `created_date_hour`: the timestap of request creation by date and hour
+Write a process to produce a time series table based on the `data/raw.csv`file we created in **Task 1** that has the following fields:
+
+- `created_date_time`: the timestap of request creation by date and hour OR just date
 - `complaint_type`: the type of the complaint
 - `count`: the count of service requests by `complaint_type` by `created_date_hour`
 
 Store this table in a csv under the `data` folder with a csv file name of your choice.
 
-### Task 3: Data Visualization
-
-Create a multi-line plot to show the total service request counts by `created_date_hour` for each `complaint_type`. Make sure you store the image of the plot in the `data` folder as a `.png` file.  
+### Python Task 3: Data Visualization
 
-### Task 4: Spatial data processing
+Create a multi-line plot to show the total service request counts by `created_date_time` for each `complaint_type`. Make sure you store the image of the plot in the `data` folder as a `.png` file.  
```

Comment:
I think having them produce multiple plots with the multiple .csv's they cached is a good test of writing reusable data viz code that sets axes/titles programmatically based on what it's passed

Reply:
I agree with Sasha especially after the work we've been doing with the QAQC app. It's great having the ability to communicate effective data viz in succinct code especially when it comes to the little formatting issues that inevitably come up